CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased](https://github.com/idealista/rsyslog_role/tree/develop)
 
+### Added
+- [#33](https://github.com/idealista/rsyslog_role/issues/33) *[FEATURE] multifunctional role* @marcelogalmor
```

Comment:
I would be a bit more explicit here with the changes you have done. i.e:
* Fix order in installing tasks because idempotency problems
* Updated tests dependencies: ansible, molecule, ...